.dockerignore

```diff
@@ -0,0 +1,2 @@
+venv310
+.python-version
```

Comment:
Add a blank line to the end of file

Reply:
there is still no blank line at the end of file